doc/Integration.rst

```diff
@@ -0,0 +1,67 @@
+Integration
+===========
+
+Doctrine integration
+--------------------
+
+Independently from your preferred framework's characteristics, it is also possible to use plain
+Doctrine with the Money VO, using its Embeddables_ feature.
+
+.. _Embeddables: http://doctrine-orm.readthedocs.org/projects/doctrine-orm/en/latest/tutorials/embeddables.html
+
+Since you should not use annotations on the third party's library files, you'll want to use the `YAML mapping`_.
```

Comment:
I am totally not familiar with the XML mapping type (see http://doctrine-orm.readthedocs.org/en/latest/reference/xml-mapping.html#example for an example); this would be a typical case where I'd point my finger to another contributor to augment this documentation when he stumbles upon this and adapts it on his own for XML.

Additionally, I tend to argue that for Doctrine mapping configuration, the usual way is Annotations / YAML / XML...